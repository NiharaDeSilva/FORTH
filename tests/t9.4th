"75.0" STR .
56 DUP CR .
Thisworks!!! STR CR .
"StringConversion!!!" STR CR .