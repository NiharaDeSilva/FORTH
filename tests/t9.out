"75.0"
56
Thisworks!!!
"StringConversion!!!"
Stack is not empty at the end of execution
Stack content:[Integer 56]